docs/expansions_and_fields.rst

```diff
@@ -8,12 +8,13 @@ Expansions and Fields
 ``expansions``
 --------------
 `Expansions`_ enable you to request additional data objects that relate to the
-originally returned List, Space, Tweets, or users. Submit a list of desired
-expansions in a comma-separated list without spaces. The ID that represents the
-expanded data object will be included directly in the List, Space, Tweet, or
-user data object, but the expanded object metadata will be returned within the
-``includes`` response object, and will also include the ID so that you can
-match this data object to the original Space, Tweet, or user object.
+originally returned List, Space, Tweets, Users, or DirectMessageEvents. Submit
+a list of desired expansions in a comma-separated list without spaces. The ID
+that represents the expanded data object will be included directly in the List,
+Space, Tweet, User, or DirectMessageEvents data object, but the expanded object
+metadata will be returned within the ``includes`` response object, and will
+also include the ID so that you can match this data object to the original
+Space, Tweet, User or DirectMessageEvents object.
```

Comment:
This section is taken from the description for the expansions parameter for endpoints that return each type of object, e.g. https://developer.twitter.com/en/docs/twitter-api/direct-messages/lookup/api-reference/get-dm_conversations-with-participant_id-dm_events for DM events.

```suggestion
originally returned List, Space, Tweets, Users, or Direct Message conversation
events. Submit a list of desired expansions in a comma-separated list without
spaces. The ID(s) that represent(s) the expanded data object(s) will be
included directly in the List, Space, Tweet, user, or event data object, but
the expanded object metadata will be returned within the ``includes`` response
object, and will also include the ID so that you can match this data object to
the original Space, Tweet, User or Direct Message conversation event object.
```

Reply:
Oh I see, the term "___ object" is used in a natural language context, and Twitter's official documents do use lower cases for "user" and "event". Also, indeed, query parameters' descriptions in all three Direct Messages lookup API references ([1](https://developer.twitter.com/en/docs/twitter-api/direct-messages/lookup/api-reference/get-dm_events) [2](https://developer.twitter.com/en/docs/twitter-api/direct-messages/lookup/api-reference/get-dm_conversations-with-participant_id-dm_events) [3](https://developer.twitter.com/en/docs/twitter-api/direct-messages/lookup/api-reference/get-dm_conversations-dm_conversation_id-dm_events)) use the term "Direct Message conversation event", though in the informal abstract/intro sections they use "Direct Messages (DM) events", "Direct Messages" and "Messages" interchangeably. 

Very accurate review! I hope this PR didn't bring more trouble than convenience...